upgrading.md

```diff
@@ -0,0 +1,202 @@
+# Upgrading (very old) WordPress
+
+Upgrading WordPress should be a simple task if its maintained and updated over time and adapted to each major version with the [requirements of that moment](https://make.wordpress.org/hosting/handbook/server-environment/).
+
+**What happens with older versions?**
+
+WordPress provides security support from WordPress 3.7 to the latest version (WordPress 6.0, __when creating this document__), each with its own database versions, PHP versions and a series of compatibilities that need to be updated over time.
+
+**What if it is not updated?**
+
+On this page there are some ideas and suggestions on how to upgrade. In many cases it will not be possible to upgrade the operating system, include new PHP versions, so will have to migrate WordPress to an updated hosting.
+
+Many assumptions will be made in this document, as each case will be different. Before doing any kind of upgrade, please make a backup of all site data: WordPress files, Database, Certificates and OS / Services configurations.
+
+IMPORTANT: This is a very manual process, not a massive one. There will be probably losses and some updates will be needed after the upgrade.
+
+[tip]In case only a major version (X.X) is indicated, use its minor (X.X.x) latest version available.[/tip]
+
+## Upgrading from WordPress 0.7 - 3.6 (by migration)
+
+Goals:
+- WordPress: upgrade to WordPress 6.0
+- PHP: upgrade to PHP 7.4
+- SQL: upgrade to MySQL 8.0 / MariaDB 10.5
```

Comment:
The versions here look to recent (going by the pattern with later migrations). Its going to be trouble if these DB versions are used and the next step attempts to downgrade them.